modmail/exts/meta.py

```diff
@@ -0,0 +1,37 @@
+import logging
+
+from discord.ext import commands
+from discord.ext.commands import Context
+
+from modmail.bot import ModmailBot
+from modmail.log import ModmailLogger
+
+log: ModmailLogger = logging.getLogger(__name__)
+
+
+class Meta(commands.Cog):
+    """Meta commands to get info about the bot itself."""
+
+    def __init__(self, bot: ModmailBot):
+        self.bot = bot
+
+    @commands.command(name="ping", aliases=("pong",))
+    async def ping(self, ctx: Context) -> None:
+        """Ping the bot to see its latency and state."""
+        await ctx.send(f"{round(self.bot.latency * 1000)}ms")
+
+    @commands.command(name="uptime")
+    async def uptime(self, ctx: commands.Context) -> None:
+        """Get the current uptime of the bot."""
+        timestamp = self.bot.start_time.format("X").split(".")[0]
+        await ctx.send(f"Start time: <t:{timestamp}:R>")
+
+    @commands.command(name="prefix")
+    async def prefix(self, ctx: commands.Context) -> None:
+        """Return the configured prefix."""
+        await ctx.send(f"My current prefix is `{self.bot.config.bot.prefix}`")
```

Comment:
Uhhhhhhh, can't we just put this in the bot's playing game status `Commands: {prefix}help`?

Reply:
No, since this is a modmail bot, the playing is typically a dm for help status. Also, the majority of server members don't need to know said prefix.